spec/importers/planning_applications_importer/address_spec.rb

```diff
@@ -99,7 +102,11 @@
       expect(ActiveJob::Base.logger).to receive(:info)
         .with(%[Planning application reference: 22/06867/PNP6A has invalid property: Full can't be blank].strip)
 
-      PlanningApplicationsImporter.new.call
+      importer
     end
   end
+
+  def importer
+    PlanningApplicationsImporter.new(local_authority_name: "lambeth").call
+  end
```

Comment:
could you have `let(:importer) { PlanningApplicationsImporter.new(local_authority_name: "lambeth").call }`?

Reply:
It should be noted that my "let" fu is weak so thank you.

I'll make the suggested change across the whole PR except the planning_application_importer_spec.rb which has an argument to the test importer function.  To get around that I would have to create a extra nested block to change the local_aurthority_name for the one test :eyeroll: